tools/compute_cloud_error.cpp

```diff
@@ -244,17 +211,17 @@ main (int argc, char** argv)
 
   // Load the first file
   pcl::PCLPointCloud2::Ptr cloud_source (new pcl::PCLPointCloud2 ());
-  if (!loadCloud (argv[p_file_indices[0]], *cloud_source))
+  if (loadPCDFile (argv[p_file_indices[0]], *cloud_source) != 0)
```

Comment:
Isn't clang tidy complaining about these unnecessary comparisons?
```suggestion
  if (loadPCDFile (argv[p_file_indices[0]], *cloud_source))
```

Reply:
It's not really unnecessary , because loadPCDFile returns an `int`, so it is mostly to increase readability (so it's clear it's not just a bool) - the compiler will optimize it. The question is: Why `PCDReader` returns an `int` and not a `bool`?

Reply:
Beats me. I have the feeling that code reviewing used to be a little more lax in the earlier stages of this project.

Reply:
There was no code review :) PCL used SVN and a number of "approved" contributors checked in directly in trunk.